Gruntfile.js

```diff
@@ -1,7 +1,40 @@
-module.exports = function(grunt) {
+
+module.exports = grunt => {
+
+  require('time-grunt')(grunt);
 
   grunt.initConfig({
 
+    clean: {
+      build: {
+        src: ['.tmp/*']
+      }
+    },
+
+    compass: {
```

Comment:
You compass is good for now.
But take sometime to look at some other configurations for the compass plugin